src/disco/components/Addon/index.js

```diff
@@ -121,31 +123,49 @@ export class AddonBase extends React.Component {
   }
 
   getThemeImage() {
-    const { getBrowserThemeData, i18n, name, previewURL } = this.props;
-    if (this.props.type === ADDON_TYPE_THEME) {
+    const { addon } = this.props;
+    const { type } = addon;
+
+    if (isTheme(type)) {
+      const { getBrowserThemeData, i18n } = this.props;
+      const { name, previewURL } = addon;
+
+      let imageUrl = getPreviewImage(addon);
+
+      if (!imageUrl && type === ADDON_TYPE_THEME) {
+        imageUrl = previewURL;
```

Comment:
Does that mean `imageUrl` can be `null` for a static theme? What should we be displaying then?

Reply:
Static themes should always have this. This was only set up for persona. Initially I was thinking after migration (of lwt themes) persona would have the previews too but it actually should change the type too and never hit here after the migration happens. If that makes sense..